src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -69,7 +69,7 @@ internal enum Verbosity
         /// <summary>
         /// Level of verbosity
         /// </summary>
-        private Verbosity verbosityLevel = Verbosity.Minimal;
+        private Verbosity verbosityLevel = Verbosity.Normal;
```

Comment:
TODO: update verbosity only for `vstest.console.exe`, waiting on spec closure for this change. Will do it in next commit in this PR.